app/config/config.yml

```diff
@@ -1,5 +1,6 @@
 imports:
     - { resource: parameters.yml }
+    - { resource: heroku.php, ignore_errors: true }
```

Comment:
Heroku consider itself production only, it does `composer install --no-dev --prefer-dist --optimize-autoloader --no-interaction` to install dependencies ([ref](https://devcenter.heroku.com/articles/php-support#installation-of-dependencies)).

This file is required if we want provide a one-click installation, otherwise some steps would be required, see https://github.com/CHH/Sylius-Heroku

That file works only on Heroku (see the initial check), is used to manage the two external services configurations in a fast way and if removed won't give errors (the `ignore_errors` option).